types/test/typescriptDocs/ModelInit.ts

```diff
@@ -21,28 +21,28 @@ interface UserCreationAttributes extends Optional<UserAttributes, "id"> {}
 
 class User extends Model<UserAttributes, UserCreationAttributes>
   implements UserAttributes {
-  public id!: number; // Note that the `null assertion` `!` is required in strict mode.
-  public name!: string;
-  public preferredName!: string | null; // for nullable fields
+  declare id: number; // Note that the `null assertion` `!` is required in strict mode.
```

Comment:
Remove `Note that the `null assertion` `!` is required in strict mode.` comment.